bftengine/src/bftengine/ReplicaBase.cpp

```diff
@@ -33,7 +33,7 @@ ReplicaBase::ReplicaBase(const ReplicaConfig& config,
       metrics_dump_interval_in_sec_(config_.metricsDumpIntervalSeconds),
       metrics_{concordMetrics::Component("replica", std::make_shared<concordMetrics::Aggregator>())},
       timers_{timers} {
-  LOG_INFO(GL, "");
+  LOG_INFO(GL, "Constuctor for replica");
```

Comment:
Are these logs required? I suggest if there is empty log info, we can remove them directly as they were not helpful till now.

Reply:
For some cases previously it helped because of method names. Since log formats have changed f these logs require description. But I agree the constructor logs can be removed. I will update changes